.github/workflows/reusable_test.yml

```diff
@@ -37,6 +37,9 @@ jobs:
       - name: Install metrics
         run: tarantoolctl rocks install metrics 1.0.0
 
+      - name: Remove external merger
```

Comment:
CI tests on Tarantool master too, so you'll need to patch this case as well, since it will start to fail after your patch merge

Reply:
I guess this is irrelevant now.